### PR TITLE
(FM-7963) Add yardoc comments to inventory_manipulation and rake_task…

### PR DESCRIPTION
**Pull request by ThoughtCrhyme**
… methods.

### CODE REVIEW

**Comment by tphoney** [comment]
could you post some screen grabs, or can i look at the resulting docs created ?

**Comment by tphoney** [comment]
> could you post some screen grabs, or can i look at the resulting docs created ?

incoming changes for the rake tasks :)

**Comment by ThoughtCrhyme** [comment]
@tphoney cleared up the warnings

File: lib/puppet_litmus/rake_tasks.rb [hunk 1]
```diff
@@ -88,6 +100,9 @@ def run_local_command(command)
     end
   end
 
+  # Provisions a list of OSes from provision.yaml file e.g. 'bundle exec rake litmus:provision_list[default]'.
+  #
+  # @param :key [Hash] key that maps to a value for a provisioner and an image to be used for each OS provisioned.
```
Comment (tphoney) [inline lib/puppet_litmus/rake_tasks.rb]: this accepts a string,

File: lib/puppet_litmus/rake_tasks.rb [hunk 2]
```diff
@@ -242,6 +271,10 @@ def run_local_command(command)
     puts 'Installed'
   end
 
+  # Provision a list of machines, install a puppet agent, and install the puppet module under test on a collection of nodes
+  #
+  # @param :key [Hash] key that maps to a value for a provisioner and an image to be used for each OS provisioned.
```
Comment (tphoney) [inline lib/puppet_litmus/rake_tasks.rb]: this is a string, not a hash

**Pull request merged**